components/rhel/rhel-ose/Vagrantfile

```diff
@@ -47,6 +47,7 @@ Vagrant.configure(2) do |config|
     config.registration.password = ENV['SUB_PASSWORD']
   end
 
+  config.vm.synced_folder '.', '/vagrant', disabled: true
```

Comment:
I am disabling the /vagrant folder sync here, since I don't know what to do to disable it in the default _Vagrantfile_ w/i the box itself. Who knows?